js/articles.js

```diff
@@ -25,10 +25,9 @@ blog.sortArticlesAuthor = function() {
   });
   for (var i=0; i<blogAuthorAlpha.length; i+=1) {
     var $authorList = $('#dropdownAuthor').clone();
```

Comment:
When you clone this, you're creating a bunch of `<option>` elements that have the same ID. ID's should be unique to a page.

![screen shot 2015-12-04 at 11 34 22 am](https://cloud.githubusercontent.com/assets/3937302/11599318/02c43836-9a7b-11e5-99ce-1bcc4b3a1f18.png)